src/middleware.ts

```diff
@@ -19,9 +19,13 @@ import sharify from "sharify"
 import siteAssociation from "artsy-eigen-web-association"
 import timeout from "connect-timeout"
 import bodyParser from "body-parser"
-import RavenServer from "raven"
 import config from "./config"
 
+// NOTE: Previoiusly, when deploying new Sentry SDK to prod we quickly start to
+// see errors like "`CURRENT_USER` is undefined". We need more investigation
+// because this only appears in prod, under load, and seems fine on staging.
+import * as Sentry from "@sentry/node"
```

Comment:
Wanted to leave this comment in place because of the issues we saw last time we tried to upgrade (which i think was 2019). Not sure if we'll still encounter problems but just to be safe.